public/styles/HomePage.scss

```diff
@@ -20,129 +20,26 @@
 
     .main {
       #primary {
-        float: left;
-        padding: 0 0 0 2em;
-        width: 78%;
 
         .home-articles-cont {
           margin: 0 0 1em;
-          overflow: visible;
-
-          .article-title {
-            border-bottom: 1px solid $color-secondary;
-            border-top: 1px solid $color-secondary;
-            color: $text-grey;
-            font-family: 'Zilla Slab';
-            font-size: 2em;
-            margin: 0 1em 0.4em 0;
-            padding: 0.5em 0;
-            text-align: center;
-            text-transform: capitalize;
-          }
+          overflow: hidden;
```

Comment:
Overflow here should be visible. This is making the cards get cut on the hover effect

Reply:
I have fixed this.